### PR TITLE
Update GitHub pages Issue #709

### PR DESCRIPTION
**Pull request by hd719**
Hello. 

cc @rickr 

Updated github templates Issue #709. 

If there is anything else I can add please do let me know! 

-Thank you.

### CODE REVIEW

**Comment by hd719** [comment]
@rickr Yes, will do so asap!

**Comment by hd719** [comment]
Hey @rickr any update on this?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
-Please use one of the following templates to create new issues:
+git <!-- Please fill out one of the sections below basd on the type of issue you're creating -->
```
Comment (rickr) [inline .github/ISSUE_TEMPLATE.md]: I think this `git` is superfluous.

**Comment by hd719** [comment]
@rickr I will fix these changes asap! Sorry about that. 

- Thank you.

**Comment by hd719** [comment]
Hey @rickr I did a rebase, but I am not sure if it worked or not. 

-Thank you.

**Pull request merged**